xbmc/platform/win32/input/IRServerSuite.cpp

```diff
@@ -76,14 +76,14 @@ void CIRServerSuite::Process()
 {
   m_profileId = CServiceBroker::GetProfileManager().GetCurrentProfileId();
   m_irTranslator.Load(IRSS_MAP_FILENAME);
+  unsigned int nRetryCount(6);
 
   // try to connect 60 times @ a 5 second interval (5 minutes)
```

Comment:
fix comment or number of retries

Reply:
comment says "60 times", but `nRetryCount` is 6. There seems to be a mismatch.